front/src/utils/DemoHttpClient.js

```diff
@@ -1,5 +1,8 @@
 export class DemoHttpClient {
   getDemoFile = async () => {
+    // Set a timer to see loaders
+    await new Promise(r => setTimeout(r, 200));
```

Comment:
I'm good with that (feels more realistic), but 200ms seems too slow on my side, it looks like the demo page is slower than the real Gladys :D 

Could you change to 100ms ?

Reply:
I will add this as an "env" config, maybe disable it by default for public mode, but with env var, I can enable it on my side.
What do you think?

Reply:
I'm good with that 👍

Reply:
Done, disabled by default, can be overridden with 
```
DEMO_MODE=true
DEMO_REQUEST_TIME=1000
```

Reply:
Works on my side!